tests/test_fedora_distro_aliases.py

```diff
@@ -40,6 +40,7 @@ def test_distro():
     assert distro.namever == "foo-123"
     assert distro.product == "fedora"
     assert distro.product == "fedora"
+    assert distro.version_number == "123"
```

Comment:
Should we return the value as a string or an int?

Reply:
Is there a chance of “floats” ( like 9.3 :) ) or leading 0s?

Reply:
Possibly. Anyway, Bodhi returns them as strings, let's return them as strings as well. To be sure.